lib/mini_sql/prepared.rb

```diff
@@ -0,0 +1,15 @@
+module MiniSql
+
+  def self.prepared?
+    Thread.current[:mini_sql_prepared_statements] == true
+  end
+
+  def self.prepared(condition = true)
```

Comment:
```ruby
builder = conn.build('select * from table where id in (?)')
builder.query(1)
builder.query([1, 2])
builder.query([1, 2, 3])
```
its creates different prepared statements

```sql
select * from table where id in ($1)
select * from table where id in ($1, $2)
select * from table where id in ($1, $2, $3)
```

no advantage perfomance to prepare for all queries if it run rarely, only for the most common cases
```ruby
ids = (1..100).to_a
builder = conn.build('select * from table where id in (?)')
MiniSql.prepared(ids.size <= 1) do 
  builder.query
end
```